docs/installation.md

```diff
@@ -23,7 +23,7 @@ java -version
 curl -fsSL get.nextflow.io | bash
 
 # Add Nextflow binary to your PATH:
-mv nextflow ~/bin
+mv nextflow ~/bin/
```

Comment:
This is part of cookiecutter - mind adding it there too please? 😁 Also _all_ the other pipelines.

Sigh. I need to write this new nf-core core code comparison tool.

Reply:
I will do that right now ;-)